.vale/styles/Vocab/OpenShiftDocs/reject.txt

```diff
@@ -0,0 +1,11 @@
+operator
```

Comment:
On my local test, I tried this but it didn't seem to work. Not sure if unsupported, or if I formatted the syntax wrong:
```suggestion
operator[s]
```

Reply:
So each line can be a valid regex. I think what you mean is `operators?`. This will pick up `operator` and `operators`. See https://regexr.com/7ebej

Reply:
Yes, exactly. I am finally gonna have to learn regex, huh? :nerd_face: That link is helpful, thanks!

Reply:
Hmm. It doesn't handle plurals easily.